community/values/src/main/java/org/neo4j/values/storable/DurationValue.java

```diff
@@ -733,6 +733,21 @@ public LongValue get( String fieldName )
         case "monthsofyear":
             val = months % 12;
             break;
+        case "monthsofquarter":
+            val = months % 3;
+            break;
+        case "quarters":
+            val = months / 3;
+            break;
+        case "quartersofyear":
+            val = (months / 3) % 4;
+            break;
+        case "weeks":
+            val = days / 7;
```

Comment:
Is this because we only know how many weeks a certain amounts of days is but not how many weeks a certain amount of months is? E.g. 1 year and 15 days will always be 54 weeks (regardless of if it is leap year or not) but we will write it as 2 weeks?